packages/pangea-sdk/pyproject.toml

```diff
@@ -19,12 +19,13 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.7.2"
 requests = "^2.27.1"
-cryptography = "37.0.2"
+cryptography = "39.0.1"
 schema = "^0.7.5"
 python-dateutil = "^2.8.2"
 alive-progress = "^2.4.1"
 pydantic = "^1.10.2"
 deprecated = "^1.2.13"
+certifi = "2022.12.7"

```

Comment:
Do we need this package? Where is it used?

Reply:
Removed certifi as a direct dependency. Will need to find what's using it, as it's showing up as having a vulnerability.